README.md

```diff
@@ -7,6 +7,23 @@ This is a [Cloud Native Buildpack](https://buildpacks.io/) that replicates the b
 
 It is written in Rust using the Cloud Native Buildpack framework [libcnb.rs](https://github.com/Malax/libcnb.rs).
 
+## Deployment
+
+### 1) Generate a release
+
+- Visit the actions page https://github.com/heroku/procfile-cnb/actions,
+- Click on "release" and then "Run workflow".
+
```

Comment:
Maybe it's worth adding a note it will release it as the version currently specified in `buildpack.toml` and it might need to be bumped prior to the release?

Reply:
Great suggestion. I'm adding a "prerelease" section and also suggesting they check the CHANGELOG which I've neglected to do.